components/developer/rust/rustc.p5m

```diff
@@ -22,8 +22,9 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-<transform file path=usr/lib/rustlib/x86_64-sun-solaris/lib/.*.so.* -> default pkg.linted.userland.action001.2 true >
-<transform file path=usr/lib/rustlib/x86_64-sun-solaris/codegen-backends/.*.so.* -> default pkg.linted.userland.action001.2 true >
+<transform file path=usr/lib/rustlib/$(RUST_ARCH)/lib/.*.so.* -> default pkg.linted.userland.action001.2 true >
+<transform file path=usr/lib/rustlib/$(RUST_ARCH)/lib/.*.so.* -> default pkg.linted.userland.action001.3 true >
+<transform file path=usr/lib/rustlib/$(RUST_ARCH)/codegen-backends/.*.so.* -> default pkg.linted.userland.action001.2 true >
```

Comment:
Please add a comment here what these transforms are about. Helps when later somebody needs to figure out what the history of these lines where. So one can evaluate if the lines are needed still or not.

Reply:
Fixed.